### DISCUSSION TITLE
Migrate away from pocketbase and performance

### DISCUSSION THREAD

**Discussion by Autumnlight02** [pocketbase/pocketbase#2107] [Q&A]
Hey, I have some questions regarding pocket base.
First of all, how many simultaneous connections can pocket base have on for example a 5€ lenode server?

Also is it possible to migrate away from pocket base later on (including password migration so all users can login as usual)
Also, what is the algorithm to decrypt the passwords? Is there a node library for that?

I really love Pocketbase, and I pitched it at my company for the early / mid-stage until we are big enough that its worth to migrate over to a good database.

**Comment by ganigeorgiev** [comment]
I haven't done recent performance tests, but you could find some details on my initial test setup in https://github.com/pocketbase/pocketbase/discussions/1629#discussioncomment-4715328 (with v0.10+ you should expect slightly better performance due to the split db pool; in the next release there will be also support for indexes management).

> Also, what is the algorithm to decrypt the passwords? Is there a node library for that?

`Bcrypt` is used for generating and verifying the password hash. Please note that it can't be decrypted. Passwords are hashed, not encrypted.


---

**If you are planning to use PocketBase for a production critical application please keep in mind that it is still under active development and backward compatibility is not guaranteed until v1 and you should be OK with reading the [CHANGELOG](https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md) and applying some manual migration steps from time to time (I try to minimize them, but sometimes are unavoidable).**
